addons/mail/__manifest__.py

```diff
@@ -87,6 +87,10 @@
         'static/src/owl/components/editable_text/editable_text.xml',
         'static/src/owl/components/emojis_popover/emojis_popover.xml',
         'static/src/owl/components/file_uploader/file_uploader.xml',
+        'static/src/owl/components/follower/follower.xml',
+        'static/src/owl/components/followers_menu/followers_menu.xml',
```

Comment:
How do you order alphabetical characters Vs special characters ? Because here you have "/" against "s", so I understand "/" is "after" "s". Is is the same for all special chars ? What is the order inside special chars ? "," is before of after "/" ?

Reply:
Alphabetical order starts from 1st character and applies on each part, i.e. substrings between `/`

e.g.:

- `static/src/owl/components/followers_menu/followers_menu.xml`
- `static/src/owl/components/follower_subtype/follower_subtype.xml`

Here is how order is decided:
- `static` vs `static` => same
- `src` vs `src` => same
- ...
- `followers_menu` vs `follower_subtype`: `_` acts like a whitespace, and "followers menu" > "follower subtype" (because "followers" > "follower").

So this is the correct order:

- `static/src/owl/components/follower_subtype/follower_subtype.xml`
- `static/src/owl/components/followers_menu/followers_menu.xml`